weed/security/tls.go

```diff
@@ -62,7 +63,22 @@ func LoadServerTLS(config *util.ViperProxy, component string) (grpc.ServerOption
 			RootProvider: serverRootProvider,
 		},
 		RequireClientCert: true,
-		VerificationType:             advancedtls.CertVerification,
+		VerificationType:  advancedtls.CertVerification,
+	}
+	options.MinTLSVersion, err = TlsVersionByName(config.GetString("tls.min_version"))
```

Comment:
`tls.min_version` config must exist?

Reply:
there're all optional, if tls.min_version not set, TlsVersionByName will return default value 0

Reply:
you're right, it's bug, and I fix it now, thank you.